core/src/main/java/io/paradoxical/cassieq/discoverable/resources/api/v1/AuthLevelRequired.java

```diff
@@ -0,0 +1,16 @@
+package io.paradoxical.cassieq.discoverable.resources.api.v1;
+
+import com.google.inject.BindingAnnotation;
+import io.paradoxical.cassieq.model.auth.AuthorizationLevel;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@BindingAnnotation
+@Target({ ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface AuthLevelRequired {
+    AuthorizationLevel[] levels();
```

Comment:
does this need to be an array? its not clear wether it needs all or one of those without looking at the code (.anyMatch in the filter)